<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchPerson</class>
 <widget class="QWidget" name="SearchPerson">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>671</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QScrollArea" name="sa_persons">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>621</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(217, 217, 217);

</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>619</width>
      <height>549</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="le_email">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;
background-color: rgb(217, 217, 217);
font: 12pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Ingrese email</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_searchPerson">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>30</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;
background-color: rgb(217, 217, 217);
color: black;

</string>
   </property>
   <property name="text">
    <string>Seguir</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
